ifrs17-template/Test/Data/ExportReportVariables.ipynb

```diff
@@ -174,7 +174,7 @@
         {
             "cell_type": "code",
             "source": [
-                "var scopesData = (await GetScopesDataAsync(args, reportStorage, Scopes)).Select(rv => rv with { Value = Math.Round(rv.Value, roundingDigits)});",
+                "var scopesData = (await GetScopesDataAsync(args, reportStorage, Scopes)).AggregateBy().Select(rv => rv with { Value = Math.Round(rv.Value, roundingDigits)});",
```

Comment:
why is this needed?

Reply:
because the import scopes are generating actuals for all novelties (N+I+C), while the report scopes are setting each one of the queried ReportVariable to Novelty C. This means at report level there are duplicates, that is, ReportVariable with the same id props but different values. Of course, those coming from actuals with Novelty N and I are zeros (default). Finally, this AggregateBy() allows to discard the duplicates (zeros) making the scopes data equal to the benchmark data that is imported from file and thus has no duplicate (because of how import works).